client/src/routes/modals/CompletedSessionModal/CompletedSessionModal.tsx

```diff
@@ -259,7 +259,7 @@ const CompletedSessionModal = () => {
           <>
             <Spacer16 />
             <Gutters>
-              <Markdown>{exercise?.description}</Markdown>
+              <ShowMoreText>{exercise?.description}</ShowMoreText>
```

Comment:
One tiny thing. Aren't we loosing the possibility to use markdown with this? Wrap it in `<ShowMoreText>`?

Reply:
Markdown doesn't seem to support `numberOfLines` or similar functionality, does it? 🤔

Reply:
ah you mean using the library, I need to try that out